package.json

```diff
@@ -113,7 +113,8 @@
     "proxy-middleware": "^0.15.0",
     "react-datepicker": "^1.4.0",
     "react-input-autosize": "^1.1.0",
-    "react-select": "1.2.1"
+    "react-select": "1.2.1",
+    "moment": "^2.18.1"
```

Comment:
This should probably be added to the peer dependencies (https://nodejs.org/en/blog/npm/peer-dependencies/) rather than the dependencies themselves.
Am I right, @yale, @SergeyPanchenkoWeWork?

Reply:
Ye I think you right, but
1) We actually don't have moment in SS we have moment-timezone
2) There are many more changes required in package.json in Plasma (Separate issue)
3) If you are at it, probably should move lodash to peer dependency as well.
4) there already moment in plasma package.json devDependencies - so need to remove it from there as well